airflow/config_templates/config.yml

```diff
@@ -2667,6 +2667,15 @@ dag_processor:
                     "repo_url": "git@github.com:example.com/my-dags.git",
                     "tracking_ref": "main",
                     "refresh_interval": 0
+            },
+            {
+                "name": "my-git-repo",
+                "classpath": "airflow.dag_processing.bundles.git.GitDagBundle",
+                "kwargs": {
+                    "subdir": "dags",
+                    "repo_url": "https://github.com/example/my-dags.git",
```

Comment:
```suggestion
```
Repo url should no longer be in this kwargs

Reply:
Why repo_url should be no longer? 45998 need to support https, doesn't it need to repo_url?

Reply:
Repo url is now configured on the connection instead of the kwargs